### ISSUE TITLE
✨ Plateforme web pour les ajouts, consultation et recherche

### ISSUE THREAD

**Issue by dofbi** [MedouneSGB/Waxtane#11]
Tout le monde n'est pas sur github, ni dev 

## waxtane.sn

Plateforme web pour les ajouts, consultation et recherche

**Comment by MedouneSGB** [comment]
Hello en effet, 
Ca serait le format le plus efficace, 
Surtout avec bon rendu mobile.

Qu'en pensez vous @daoodaba975  @ManuSquall

**Comment by ManuSquall** [comment]
Effectivement ce serait intéressant, certains sites se basent également sur des fichiers markdown pour afficher leur contenu!

**Comment by daoodaba975** [comment]
Yes effectivement comme l'a dit Manu, on peut travailler sur un projet qui fetch directement le fichier Markdown.

**Comment by MedouneSGB** [comment]
Cool ça et pour les Pull Request ? 
Comment ça se passe quand un utilisateur ajoute un enregistrement depuis le site.

**Comment by daoodaba975** [comment]
Ok pour faudra que je fasse des recherches sur ça, j'ai pas encore testé ça.

**Comment by dofbi** [comment]
Salaam Team 🙏🏾 
Voici un début d'intégration avec [Astro](https://docs.astro.build/en/getting-started/)

### Voici une démo
https://codesandbox.io/s/waxtane-7zr15x

**Comment by MedouneSGB** [comment]
Hello Master, super !
J'étudie les intégrations !!✨

**Comment by daoodaba975** [comment]
Great!
I'm going to check it.

**Comment by MedouneSGB** [comment]
Hello guys 😁, 
Le dernier Pull Request était 🔥
J'ai ajouté le lien vers Netlify https://waxtane.netlify.app/

**Comment by MedouneSGB** [comment]
Hi guys,

- J'ai un peu réarranger le répertoire principal en déplaçant les composants du site dans le dossier web/

- Aussi ajouté un favicon transparent.
 678343c19b3cddaa4087c7fe55ac49af94bb1df0

**Comment by Dar-rius** [comment]
Hi @MedouneSGB et @dofbi, je voulais savoir si les visiteurs de la plateforme qui voudront contribuer en ajoutant une phrase devront le faire dans le fichier waxtane.csv ?

**Comment by MedouneSGB** [comment]
Hey @Dar-rius, 
Yep pour le moment les ajouts se font sur [waxtane.csv](https://github.com/MedouneSGB/Waxtane/blob/f3dd507bfe0e4cd618c0f2355f90f92df952ac23/data/waxtane.csv).

On aimerait ajouter la possibilité de le faire depuis le site.

**Comment by Dar-rius** [comment]
j'ai essayer avec la librairie "csv-writer" mais j'ai eu un soucis avec la fonction 'writerFile' de fs.
voici le message d'erreur qui m'envoie:  
'writeFile' is not exported by __vite-browser-external, imported by ../node_modules/csv-writer/src/lib/file-writer.ts

**Comment by MedouneSGB** [comment]
Pas encore ! 😶
Continuons les recherches.
